.circleci/docker/common/install_rocm.sh

```diff
@@ -44,9 +37,23 @@ install_ubuntu() {
       DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated ${MIOPENKERNELS}
     fi
 
-  # Cleanup
-  apt-get autoclean && apt-get clean
-  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+    # "install" hipMAGMA into /opt/rocm/magma by copying after build
+    git clone https://bitbucket.org/icl/magma.git -b hipMAGMA
+    pushd magma
+    cp make.inc-examples/make.inc.hip-mkl-gcc make.inc
+    echo 'LIBDIR += -L$(MKLROOT)/lib' >> make.inc
+    echo 'LIB += -Wl,--enable-new-dtags -Wl,--rpath,/opt/rocm/lib -Wl,--rpath,$(MKLROOT)/lib -Wl,--rpath,/opt/rocm/magma/lib' >> make.inc
+    echo 'DEVCCFLAGS += --amdgpu-target=gfx803 --amdgpu-target=gfx900 --amdgpu-target=gfx906 --amdgpu-target=gfx908' >> make.inc
```

Comment:
Hmm, can we infer supported gpu targets from environment variable or something?

Reply:
We don't currently have any way of querying the default set of targets for a particular ROCm release.  Today, every ROCm package is specifying their own set of gfx targets, though we do coordinate across projects to ensure they match.  [Example from HIP](https://github.com/ROCm-Developer-Tools/HIP/blob/2080cc113a2d767352b512b9d24c0620b6dee790/CMakeLists.txt#L370).

This PR matches that current list.  As we upgrade ROCm releases and prepare PRs for new ROCm versions, this list can change.

We're also discussing internally whether we can provide a hipMAGMA package alongside our other ROCm packages.  Such a package doesn't exist yet, so we're forced to build from source today.